rpc/documentation/api.md

```diff
@@ -1417,6 +1417,21 @@ transaction was seen.
 - `int64 timestamp`: The Unix time of the earliest time this transaction was
   seen.
 
+  **Nested enum:** `TransactionType`
+
+  - `REGULAR`: All normal regular tx tree transactions.
+
+  - `TICKET_PURCHASE`: A transaction that purchases a ticket to be consumed later
+    by a vote transaction upon random selection.
+
+  - `VOTE`: A transaction that uses a ticket purchase transaction as an input 
+    and receives a stake base reward as one of the outputs.
+  
+  - `REVOCATION`: A transaction that will free a previously locked utxos in 
+    a ticket purchase due to expiry or the ticket being missed.
```

Comment:
Since the previous talks about stake bases, we should probably mention those here as well.  Coins created by revocations still must reach maturity, even though there is no reward for the vote.